style/extensions.txt

```diff
@@ -866,7 +867,7 @@ added to ename:VkObjectType (as defined in the "`Debugging`" section of the
 identify and track objects of the new type.
 
 The new enumeration value must conform to the naming defined in the
-<<extension-enumerant-names,Extension Enumerant Names>> section.
+<<_extension_enumerant_names,Extension Enumerant Names>> section.
```

Comment:
What's an "implicit anchor"? I noticed some anchors start with `_`. I have not found any system to this, and not sure what the deal is...

Reply:
oh, nvm.

Reply:
If you don't precede the section with an explicit [[extension-enumerant-names]] (or other anchor name), it will AFAICT generate one by simplifying the section title and preceding with an _. I don't know exactly where this is documented as we rely on explicit anchors - this was an oversight.

Reply:
yea, so I figured; I previously only inspected anchors in browser...